src/main/java/com/xceptance/xlt/agentcontroller/TestResultAmount.java

```diff
@@ -22,7 +22,11 @@
 {
     ALL("Measurements, result browser data, and logs [all]", "1"),
     MEASUREMENTS_AND_RESULTBROWSER("Measurements and result browser data", "2"),
+    MEASUREMENTS_AND_LOGS("Measurements and logs", "3"),
     MEASUREMENTS_ONLY("Measurements only", "3"),
```

Comment:
The key "3" is already used. Please ensure all keys are unique.

Reply:
Fixed.